### PR TITLE
Anime Search: Update and restyle code

### PR DESCRIPTION
**Pull request by SilverTactic**
Knowing me, you'd probably wanna test the script out yourself. But otherwise, it seems to work fine for me
Updates include:
1) No longer uses an animecache file (you can delete that file)
2) Prevents hentai anime from being broadcasted in non-private rooms

### CODE REVIEW

File: chat-plugins/animesearch.js [hunk 1]
```diff
@@ -89,9 +79,15 @@ exports.commands = {
 		if (this.broadcasting && room.id === 'lobby') return this.errorReply("This command cannot be broadcasted in the Lobby.");
 		if (!target || !target.trim()) return this.sendReply('/' + cmd + ' [query] - Searches for an anime based on the given search query.');
 
-		search(target.trim()).then(function (response) {
-			this.sendReplyBox(response);
-			if (this.broadcasting) room.update();
-		}.bind(this));
+		searchAnime(target.trim()).then((anime) => {
+			let genres = anime.genres.map(function (genre) { return toId(genre.name) });
+			if (!room.isPrivate && (genres.indexOf('hentai') > -1 || (genres.indexOf('yaoi') > -1 && anime.title.match(/pico/ig))) && this.broadcasting)
```
Comment (panpawn) [inline chat-plugins/animesearch.js]: Let's just prevent hentai from being able to be searched with this entire script regardless of room, please.

**Comment by SilverTactic** [comment]
Alright, I can do that
The issue: I'm still having trouble melding/updating PRs (I can't seem to switch to my branch). Would it be possible for you to meld my pulls, if I make another one over here to fix the issue?

**Comment by panpawn** [comment]
What if you do this the really lazy way and just click 'edit' on the top of the PR page on GitHub's site, and then I'll just manually merge, @SilverTactic ?

**Comment by SilverTactic** [comment]
That edits the PR's title lol
Even if I pressed the pencil icon on the PR tab, it'd just create a new pull on the same branch, so they'd have to be melded together

**Pull request merged**